app/models/concerns/delegate_attribute.rb

```diff
@@ -0,0 +1,54 @@
+module DelegateAttribute
```

Comment:
`delegate` doesn't work like `delegate_attribute` does, and talked to @jmileham and we are fine including this module here for now.